app/controllers/user_controller.py

```diff
@@ -7,6 +7,14 @@
 user_routes = Blueprint('user_routes', __name__)
 
 
+@user_routes.route('/', methods=['GET'])
+@login_required
+def get_feed(user_id: int) -> Any:
+    feed = user_service.get_feed(user_id)
+
+    return render_template('feed.html', feed=feed)
```

Comment:
Need logged in user object specifically for full name. Full name is not stored in the session object.

Reply:
what do you need the full name for?

Reply:
Sidebar section of the feed (see attached image).

![image](https://user-images.githubusercontent.com/15861967/109752516-8b81d600-7bae-11eb-9003-6eb1720c1fbc.png)